<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GwmOpenXYEventLayerDialog</class>
 <widget class="QDialog" name="GwmOpenXYEventLayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="filenameLabel">
       <property name="text">
        <string>File name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="layerNameLabel">
       <property name="text">
        <string>Layer name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="encodingLabel">
       <property name="text">
        <string>Encoding</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="fileNameEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="fileNameOpenFileDialogBtn">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="encodingCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="layerNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>File Format</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QRadioButton" name="formatCSVRadio">
          <property name="text">
           <string>CSV (Comma Separated Values)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="formatRegRadio">
          <property name="text">
           <string>Regular expression delimiter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="formatCustomRadio">
          <property name="text">
           <string>Custom delimiters</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formatRegSettingLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Expression</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="formatExpressionEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formatCustomSettingLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Delimiter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="1">
           <widget class="QCheckBox" name="formatCustomDelimiterSemicolonRadio">
            <property name="text">
             <string>Semicolon</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="formatCustomDelimiterColonRadio">
            <property name="text">
             <string>Colon</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="formatCustomDelimiterSpaceRadio">
            <property name="text">
             <string>Space</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="formatCustomDelimiterTabRadio">
            <property name="text">
             <string>Tab</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="formatCustomDelimiterCommaRadio">
            <property name="text">
             <string>Comma</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <layout class="QFormLayout" name="formLayout_6">
            <item row="0" column="1">
             <widget class="QLineEdit" name="formatCustomDelimiterOthersEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="formatCustomDelimiterOthersRadio">
              <property name="text">
               <string>Others</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Quote</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="formatCusomQuoteEdit">
          <property name="text">
           <string>&quot;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Escape</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="formatCustomEscapeEdit">
          <property name="text">
           <string>&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Record and Fields Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="1" column="1">
       <widget class="QCheckBox" name="recordTrimFields">
        <property name="text">
         <string>Trim fields</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="recordDecimalSepComma">
        <property name="text">
         <string>Decimal separator is comma</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Number of header lines to discard</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="recordNumberDiscardLinesSpin"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="recordFirstLineAsFieldNames">
        <property name="text">
         <string>First record has field names</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="recordDetectFieldTypes">
        <property name="text">
         <string>Detect field types</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="recordDiscardEmptyFields">
        <property name="text">
         <string>Discard empty fields</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Geometry Definition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Z field</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="geometryYField"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y field</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X field</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="geometryXField"/>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="geometryZField"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>M field</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QComboBox" name="geometryMField"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="geometryDMSCoord">
        <property name="text">
         <string>Degree/Minute/Second coordinates</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="geometryCRSCombo"/>
          </item>
          <item>
           <widget class="QToolButton" name="geometryOpenCRSDialogBtn">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Geometry coordnate reference system</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GwmOpenXYEventLayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GwmOpenXYEventLayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
